recipe/conda_build_config.yaml

```diff
@@ -390,6 +390,7 @@ python:
   - 2.7
   - 3.5
```

Comment:
Could we drop Python 3.5?

Reply:
Not now.

Reply:
Why not?

Reply:
Given the other builds needed for the new compilers, I don't think it is reasonable to hang on to Python 3.5 when adding Python 3.7.